bundles/CoreBundle/Resources/translations/de.json

```diff
@@ -435,6 +435,7 @@
     "columns": "Spalten",
     "children_grid": "Kinder Tabelle",
     "only_children": "Nur direkte Kindelemente",
+    "only_unreferenced": "Nur nicht referenzierte",
```

Comment:
Please revert this change, only EN is maintained on GitHub, all other languages are on POEditor :)

Reply:
Ok, reverted this. I do not know anything about PO editor so I cannot add the non-english translations with it. Can this be done by somebody else?